src/services/ai-server/ai-persona-service/ai.persona.service.service.ts

```
@@ -6,17 +6,17 @@ import { EntityNotFoundException } from '@common/exceptions';
 import { AuthorizationPolicy } from '@domain/common/authorization-policy';
 import { AiPersonaService } from './ai.persona.service.entity';
 import { IAiPersonaService } from './ai.persona.service.interface';
-import { CreateAiPersonaServiceInput as CreateAiPersonaServiceInput } from './dto/ai.persona.service.dto.create';
-import { DeleteAiPersonaServiceInput as DeleteAiPersonaServiceInput } from './dto/ai.persona..service.dto.delete';
+import {
+  CreateAiPersonaServiceInput as CreateAiPersonaServiceInput,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)